gym-service/src/test/features/trainee/trainee.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Trainee: Profile Management
+
+  Scenario: Successfully access profile
+    Given the trainee is registered with username: "testTrainee"
+    When the trainee requests the trainee profile
+    Then the response status for trainee should be 200
+    And the response should contain the trainee's profile information:
+      | firstName | lastName | dateOfBirth | trainerList | userIsActive | address |
+
+  Scenario: Access profile without JWT
+    Given the trainee is registered with username: "testTrainee"
+    When the trainee requests the trainee profile
+    Then the response status for trainee should be 403
```

Comment:
just note, this scenario should return 401 Unauthorized since no authorization was provided. Broken or expired token is also 401. 403 Forbidden usually when provided authorization does not have permissions for request